music21/expressions.py

```diff
@@ -1027,42 +1030,97 @@ class Turn(Ornament):
     A turn or Gruppetto.
 
     * Changed in v7: size is a Generic second.  removed unused nachschlag component.
+    * Changed in v9: Added support for delayed vs non-delayed Turn.
     '''
-    def __init__(self, **keywords):
+    def __init__(self, delay: OrnamentDelay | OffsetQL = OrnamentDelay.NO_DELAY, **keywords):
```

Comment:
If anything needs to be changed, I'd prefer this to be keyword only:  `self, *, delay: ...` -- if no other changes needed, then it's good.

Reply:
Done.